shell/TestPackage.ps1

```diff
@@ -0,0 +1,24 @@
+$ErrorActionPreference = "Stop"
+$LastExitCode = 0
+$env:PATH +=";$($env:SystemDrive)\\ProgramData\\chocolatey\\bin"
+
+# https://github.com/chocolatey/choco/issues/512
+$validExitCodes = @(0, 1605, 1614, 1641, 3010)
+
+$package  = ls c:\packages\*.nupkg
+if (!$package) { $package = ls c:\packages\remote\*.nupkg; $remote=$true }
+if (!$package) { Write-Host 'No packages found in directory packages or packages\remote'; return }
```

Comment:
So why did you have 2 choco install lines there and asking to choose one ? I added remote just because it was there.